.github/workflows/nerdctl.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
       - name: Export logs
         if: always()
         run: |
-          sudo cat /etc/cni/net.d/* 
+          sudo find /etc/cni/net.d/ -type f -exec sh -c 'echo "{}" && cat "{}"' \;
```

Comment:
?

Reply:
I have explained here https://github.com/kubernetes-sigs/kind/pull/3850#issuecomment-2600945612 that the previous method would lead to errors.

failed job: https://github.com/kubernetes-sigs/kind/actions/runs/12855249247/job/35840567917